Userland/Applications/Presenter/Presentation.cpp

```diff
@@ -129,20 +130,33 @@ ErrorOr<NonnullOwnPtr<Presentation>> Presentation::load_from_file(StringView fil
     if (maybe_metadata.is_null() || !maybe_metadata.is_object() || maybe_slides.is_null() || !maybe_slides.is_array())
         return Error::from_string_view("Metadata or slides in incorrect format"sv);
 
+    auto const& maybe_templates = global_object.get("templates"sv);
+    if (!maybe_templates.is_null() && !maybe_templates.is_object())
+        return Error::from_string_view("Templates are not an object"sv);
+    auto json_templates = maybe_templates.is_null() ? JsonObject {} : maybe_templates.as_object();
```

Comment:
It feels a little awkward to assign this to an empty JsonObject, and then try and read its members, when you know it doesn't have any. I think it would be better to wrap the `try_for_each_member()` call below in an `if (!maybe_templates.is_null())` instead.

Reply:
I guess this time I actually wanted to avoid rightwards drift...